### PR TITLE
NGI final!

### PR DESCRIPTION
**Pull request by guillermo-carrasco**
I also restructured slightly the `README.md` so that its coherent with the presentation. 

I may want to change some bits of texts when I start practicing it, but you can merge this if you want to, its mainly finished! :smile:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,29 +46,17 @@ They will also get a peek behind the website that allows clinicians to
...
 
-2. Scout, our web-based data delivery interface (10 min)
-  1. From Excel sheet to Flask website
-  2. From JavaScript-mess to Python-bliss
+The content of the presentation is structured into [3 different parts](/parts).
+On each part there is a `slides.md` file that accompany the slides for that part,
+which are located in the same directory under the name `presentation.key`
 
-**Finale**
+The complete slides set is located [here](/presentation.key).
```
Comment (guillermo-carrasco) [inline README.md]: this needs to be added when we merge the slides :)

File: README.md [hunk 2]
```diff
@@ -1,4 +1,6 @@
-# Proposal sketch for PyCon Sweden 2015
+# Presentation material for PyCon Sweden 2015
+
+![PyCon](/assets/logo_pycon.png?raw=True)
```
Comment (robinandeer) [inline README.md]: Congrats - you're thinking visually ^^

File: parts/2-ngi_pipeline/slides.md [hunk 3]
```diff
@@ -1,5 +1,27 @@
...
+
+### How is data generated?
+
+_DISCLAIMER: Oversimplified!_
+
+Someone (researcher/doctor) takes a sample of the organism/tissue they want to study,
+they send it to us and, after some preparation in the laboratory we place the sample in the machine
+for sequencing. The machine converts the sample into binary information readable by a
+computer, in where after some processing, we obtain the "famous" A-C-T-G strings. The
+amount of data generated (deepnes of sequencing/coverage) is determined by both
```
Comment (robinandeer) [inline parts/2-ngi_pipeline/slides.md]: s/deepnes/depth

Comment (robinandeer) [inline parts/2-ngi_pipeline/slides.md]: Perhaps say something like

> the more you can afford to sequence the better the quality of the final output

eller?

File: parts/2-ngi_pipeline/slides.md [hunk 4]
```diff
@@ -17,17 +39,26 @@ it arrived to the analysis cluster and has been long-term archived before removi
 In the early times (this is about 3 years ago...) this whole process was done manually,
 this is nowadays impossible.
 
-_NOTE: A slide with a flowchart for TACA will go with this explanation_
 
 **TACA - Tool for the Automated Cleanup and Analysis** is a tool completely written
-in Python 2.7 that does precisely this. When a machine has finished sequencing, TACA
+in Python 2.7 that does precisely this among other stuff. When a machine has finished sequencing, TACA
```
Comment (robinandeer) [inline parts/2-ngi_pipeline/slides.md]: If you're going to _say_ this I think it just adds confusion - no one _really_ cares so cut corners and simplify :smile:

File: parts/2-ngi_pipeline/slides.md [hunk 5]
```diff
@@ -43,19 +74,20 @@ RabbitMQ + Celery, is that the HPC has its own task scheduler, called SLURM, tha
 will allocate and distribute the jobs among computing nodes.
 
 In the HPPT request received in the server, the id of the dataset to be analyzed is
```
Comment (robinandeer) [inline parts/2-ngi_pipeline/slides.md]: s/HPPT/HTTP/

File: parts/2-ngi_pipeline/slides.md [hunk 6]
```diff
@@ -69,6 +101,15 @@ needed _only_ to check the final results, but all the repetitive and tedious tas
...
 automatically.
 
+This whole automation has been a huge improvement in our workflow, allowing us to
+be more and more ambitious in terms of how many projects we accept and how many samples
+per year we want to sequence and analyze.
+
+In the core facility where I work, we only do research, and in that case its okay to
+give the researchers all the output of our pipeline so that they can work woth the
+results. However on the clinical side the story is a bit different, and this is what
+Robin will tell you right now :)
```
Comment (robinandeer) [inline parts/2-ngi_pipeline/slides.md]: That's me :fearful:

**Comment by robinandeer** [comment]
When you show the Click code for ``entry_points`` write also that this is specified in ``setup.py``

**Comment by robinandeer** [comment]
You have a tonne of code in your slides!! Now I feel bad :disappointed:

File: parts/2-ngi_pipeline/slides.md (hunk 3 above)
Comment (guillermo-carrasco) [inline parts/2-ngi_pipeline/slides.md]: hmm yep, sounds better, tack! 

(deepnes.. :sweat_smile: , sorry)

File: parts/2-ngi_pipeline/slides.md (hunk 4 above)
Comment (guillermo-carrasco) [inline parts/2-ngi_pipeline/slides.md]: :+1:

File: README.md (hunk 2 above)
Comment (guillermo-carrasco) [inline README.md]: woha!

File: parts/2-ngi_pipeline/slides.md (hunk 6 above)
Comment (guillermo-carrasco) [inline parts/2-ngi_pipeline/slides.md]: yes! :smile: :+1:

**Comment by guillermo-carrasco** [comment]
No don't, come on! I'll pass the code slides quite fast, is just so that developers get their drug :smile: 

Thanks for the review, committed the changes :)

**Comment by robinandeer** [comment]
You added this to the presenter notes in the intro presentation right:

> Actually, there are some analogies with how computers and our bodies work.

Nothing more?

**Comment by robinandeer** [comment]
Rebase my old padawan!

Select my version of the .key-file and then you can add back the presenter note, sí?

**Comment by guillermo-carrasco** [comment]
Done :-)

**Comment by guillermo-carrasco** [comment]
And I still find rebasing very confusing :sweat_smile: ... master

**Comment by robinandeer** [comment]
Yeah, it can get make you bewildered at times but I think it makes you understand Git better :smile:

**Pull request merged**